geode-core/src/integrationTest/java/org/apache/geode/internal/SSLConfigJUnitTest.java

```diff
@@ -153,6 +156,66 @@ public static void initializeSSLMaps() {
   @After
   public void tearDownTest() {}
 
+  @Test
+  public void slowerReceiverShouldThrowExceptionWhenEnabledClusterTLS() throws Exception {
+    Properties props = new Properties();
+    props.setProperty(SSL_ENABLED_COMPONENTS, "cluster");
+    props.setProperty(ASYNC_DISTRIBUTION_TIMEOUT, "1");
+
+    try {
```

Comment:
I recommend using AssertJ catchThrowable for these expected exceptions. It would end up looking like:
```
import static org.assertj.core.api.Assertions.assertThat;
import static org.assertj.core.api.Assertions.catchThrowable;

Throwable thrown = catchThrowable(() -> new DistributionConfigImpl(props));
assertThat(thrown)
  .isInstanceOf(IllegalArgumentException.class)
  .hasMessageContaining("async-distribution-timeout greater than 0 is not allowed with cluster TLS/SSL.");
```

Reply:
Fixed.